### PR TITLE
[3.2.x] Pinned asgiref dependency for 3.2 release series.

### PR DESCRIPTION
**Pull request by carltongibson**

### CODE REVIEW

**Review by felixxm — APPROVED** [review]
@carltongibson Thanks :+1: I would do the same in `tests/requirements/py3.txt` and https://github.com/django/django/blob/29e2df24e7cb43c3600b2fb45472016709319a5b/docs/internals/contributing/writing-code/unit-tests.txt#L274

File: docs/releases/3.2.txt [hunk 1]
```diff
@@ -739,6 +739,9 @@ Miscellaneous
   messages in the :rfc:`6265` compliant format. Support for cookies that use
   the old format remains until Django 4.1.
 
+* The minimum supported version of ``asgiref`` is increased from 3.2 to
```
Comment (felixxm) [inline docs/releases/3.2.txt:742]:
```suggestion
* The minimum supported version of ``asgiref`` is increased from 3.2.10 to
```

**Comment by carltongibson** [comment]
Closing in favour of #14223.

File: tests/requirements/py3.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-asgiref >= 3.2.10
+asgiref >= 3.3.2
```
Comment (felixxm) [inline tests/requirements/py3.txt:1]: CI will check this branch until 2024 :smile: , so ..
```suggestion
asgiref >=3.3.2,<4
```

**Pull request closed without merging**